pink/utils.py

```diff
@@ -59,6 +59,23 @@ def get_root_joint_dim(model: pin.Model) -> Tuple[int, int]:
     return 0, 0
 
 
+def get_joint_idx(model: pin.Model, joint_name: str) -> Tuple[int, int]:
+    """get joint index in tangent space
+
+    Args:
+        model: Robot model.
+
+    Returns:
+        nq: Number of configuration dimensions.
+        nv: Number of tangent dimensions.
+    """
+    if model.existJointName(joint_name):
+        joint_id = model.getJointId(joint_name)
+        joint = model.joints[joint_id]
+        return joint.idx_q, joint.idx_v
+    raise "cannot find the joint index(q, v) corresponding joint index"
```

Comment:
```suggestion
    raise PinkError(f"cannot find the joint index corresponding to joint {joint_name}")
```
``PinkError`` is defined in ``pink.exceptions``.